src/devices/Mfrc522/README.md

```diff
@@ -17,8 +17,8 @@ SpiConnectionSettings connection = new(0, 1);
 // Here you can use as well MfRc522.MaximumSpiClockFrequency which is 10_000_000
 // Anything lower will work as well
 connection.ClockFrequency = 5_000_000;
-SpiDevice spi = SpiDevice.Create(connection)
-mfrc522 = new(spi, 4);
+SpiDevice spi = SpiDevice.Create(connection);
+MfRc522? mfrc522 = new(spi, 4);
```

Comment:
should this be `MfRc522 mfrc522 = new(spi, 4);` ?

Reply:
I was following the sample program as a guideline